src/main.rs

```diff
@@ -245,8 +282,48 @@ impl FuzzProject {
             cmd.arg(self.corpus_for(&target)?);
         }
 
-        exec_cmd(&mut cmd).chain_err(|| format!("could not execute command: {:?}", cmd))?;
-        Ok(())
+        if jobs == 1 {
+            exec_cmd(&mut cmd).chain_err(|| format!("could not execute command: {:?}", cmd))?;
+            Ok(())
```

Comment:
could `return` here if you wanted to avoid the indentation in the `else` block, up to you